.storybook/blocks/ComponentDetails.jsx

```diff
@@ -318,119 +306,76 @@ function fetchNpmData(packageName, setnpmData, setIsLoading) {
-	
-	if (linkType === "package") {
-		// NPM package name and link 
-		packageName = packageJson?.name ?? undefined;
-		packageLink = (packageName && typeof packageName !== "undefined") ? `https://npmjs.com/${packageName}` : false;
-	}
-	
-	else if (linkType === "repository") {
-		// repo name and link 
-		packageName = packageJson?.name ? packageJson?.name.split('/').pop() : undefined;
-		packageLink = (packageName && typeof packageName !== "undefined") ? `https://github.com/adobe/spectrum-css/tree/main/components/${packageName}` : false;
+const fetchLogo = (brand) => {
+	switch(brand) {
+		case "npm":
+			return NpmSVG;
+		case "GitHub":
+			return GitHubSVG;
+		case "Adobe":
+			return AdobeSVG;
 	}
 
-	else if (linkType === "guidelines") {
-		// checks if the any nested component name is included in the page URL
-		for(let i = 0; i < packageJson?.spectrum?.length; i++) {
-			if (window.location.href.includes(packageJson?.spectrum[i]?.componentName)) {
-
-				// guidelines site name and link
-				packageName = packageJson?.spectrum[i]?.componentName ?? undefined;
-				packageLink = (packageName && typeof packageName !== "undefined") ? packageJson?.spectrum[i]?.guidelinesLink : false;
-				break;
-			} 
-			packageName = packageJson?.spectrum[i]?.componentName ?? undefined;
-			packageLink = (packageName && typeof packageName !== "undefined") ? packageJson?.spectrum[i]?.guidelinesLink : false;
-		}
-	}
+	return;
+}
 
-	else {
-		console.warn(`Are you sure you mean "${linkType}"? Please use a valid link type instead: "package", "repository", "guidelines"`);
-	}
+
+export const ResourceLinkContent = ({ heading, alt, logo, href }) => {
+	if (!href) return;
 
 	return (
-		<>
-			{packageLink ? <>
-					<ResourceLink href={packageLink} rel="noopener" className="sb-unstyled">
-						{iconSvgs(linkType)}
-						<ResourceTextWrapper>
-							<div className="spectrum-Heading spectrum-Heading--sizeXS">View {linkType}</div>
-							<div className="spectrum-Body spectrum-Body--sizeS">{convertLinkTypeText(linkType)}</div>
-						</ResourceTextWrapper>
-					</ResourceLink>
-				</>
-				: ""
-			}
-		</>
-	)
+		<ResourceLink href={href} className="sb-unstyled" title={alt}>
+			<ResourceIconWrapper dangerouslySetInnerHTML={{ __html: fetchLogo(logo) }} />
+			<ResourceTextWrapper>
+				{heading ? <Heading size="xs">{heading}</Heading> : ""}
+				{alt ? <Body size="s">{alt}</Body> : ""}
+			</ResourceTextWrapper>
+		</ResourceLink>
+	);
 };
 
-export const ResourceListDetails = ({data}) => {
-	const packageJson = data;
+export const ResourceListDetails = ({ packageName, spectrumData = [] }) => {
+	if (!packageName) return;
+
+	let href;
+	// checks if the any nested component name is included in the page URL
+	for(let i = 0; i < spectrumData?.length; i++) {
+		if (
+			spectrumData[i]?.guidelines &&
+			(
+				window.location.href.includes(spectrumData[i]?.componentName) ||
+				window.location.href.includes(spectrumData[i]?.componentName.replace("-", ""))
```

Comment:
Is this the same `componentName` that was in the components' `package.json`? It looks like that is removed and the Docs pages aren't loading anymore.

Reply:
I added a couple back in in this commit https://github.com/adobe/spectrum-css/pull/3129/commits/60d92b5268f75916d3e65ecf7214341ce4beece0 just to see if the pages would render. Those are also the nested components (field label + form; progress bar + meter), and I believe they're rendering and working as I expected them to. Form doesn't render a guidelines resource card, and meter/progressbar/fiedlabel all render their own guidelines resource card that links to their pages appropriately.

I'm unsure of a way to do it without adding a `componentName` to the package.jsons. I thought maybe we could use the `packageName`, but I couldn't get that to work. All of our urls put hyphens between the words ie. field-label, whereas our packageNames do not ie. fieldlabel.

@castastrophe I'm open to ideas! If we like this approach, I can add `componentName`s back to the package.jsons.

Reply:
Oh! Here's another approach (thanks goes to Josh for the push in this direction): https://github.com/adobe/spectrum-css/pull/3129/commits/0592536e5d54968b67c6564ded21126ddeb34bd9

By declaring `docsPage: false` on the form, I was able to avoid that form docs page from rendering a link for the field label guidelines (which we were trying to avoid). Because `docsPage` _is_ undefined for all other stories, the rest of the components that still have `guidelines` defined, render their links as before. 

This might be a slightly simpler fix than going through each package.json to add `componentName` back in.

Reply:
Is it necessary to add the `hasDocsPage` variable? Can't this already be determined by whether the key `guidelines` is undefined or empty in the package JSON? It looks like that is already being checked with `spectrumData[i]?.guidelines`.